src/LoginScreen.tsx

```diff
@@ -3,16 +3,22 @@ import React from "react";
 function loginScreen() {
   return (
     <>
-      <h1>Bem-vindo(a) à Taqtile!</h1>
-      <div>
-        <text>E-mail</text>
-        <input />
-      </div>
-      <div>
-        <text>Senha</text>
-        <input />
-      </div>
-    </>
+      <h1>
+        Bem-vindo(a) à Taqtile!
+      <h1/>
+      </h1>
+    <form>
+      <label>
+        E-mail
+      </label>
+      <input type="text"/>
```

Comment:
Para fins de acessibilidade é melhor:
```suggestion
      <label>
          E-mail
         <input type="text"/>
      </label>
      
```